_posts/zh/newsletters/2023-08-02-newsletter.md

```diff
@@ -0,0 +1,80 @@
+
+- [Core Lightning #6399][]添加了对 `pay` 命令的支持，用于支付本地节点创建的发票。这可以简化在后台调用 CLN 的软件的帐户管理代码，正如最近的[邮件列表主题][fiatjaf custodial]中所讨论的那样。{% assign timestamp="33:03" %}
+
+- [Core Lightning #6389][]添加了一个可选的 CLNRest 服务，“一个基于 Python 的轻量级 Core Lightning 插件，可将 RPC 调用转换为 REST 服务。通过生成 REST API 端点，它可以在幕后执行 Core Lightning 的 RPC 方法并提供 JSON 格式的响应”。有关详细信息，请参阅[文档][clnrest doc]。{% assign timestamp="35:48" %}
+
+- [Core Lightning #6403][]和[#6437][core lightning #6437]将 runes 授权和身份验证机制从 CLN 的 commando 插件(见[周报#210][news210 commando])移至其核心功能中，允许其他插件使用它们；创建、销毁和重命名 runes 相关的几个命令也已更新。（译者注：runes 是 CLN 定制的简化版 macaroons 协议。）{% assign timestamp="37:37" %}
+
+- [Core Lightning #6398][]使用新的 `ignorefeelimits` 选项扩展了 `setchannel` RPC，该选项将忽略通道的最低链上费用限制，允许远程通道对手方将费用设置为低于本地节点允许的最低费用。这可以帮助解决另一个 LN 节点实现中的潜在错误，或者可以被用来消除部分可信通道作为问题根源的费用争用问题。{% assign timestamp="39:52" %}
+
+- [Core Lightning #5492][]添加了“用户级静态定义跟踪点(USDT)”以及使用它们的方法。这些允许用户探测其节点的内部操作以进行调试，而在不使用跟踪点时不会引入任何显著的开销。有关之前将 USDT 支持纳入 Bitcoin Core 的信息，见[周报#133][news133 usdt]。{% assign timestamp="45:52" %}
+
+- [Eclair #2680][]添加了对[BOLTs #863][]中提出的[通道拼接协议][topic splicing]所需的静止协商协议的支持。静止协议防止共享通道的两个节点相互发送任何新的[HTLCs][topic htlc]，直到某个操作完成为止，例如就拼接参数达成一致并合作签署链上拼接输入或拼接输出的交易。在拼接协商和签名期间收到的 HTLC 可能会使之前的协商和签名无效，因此更简单的是，只需暂停 HTLC 中继以完成拼接交易所必需的共同签名的几次网络往返通信。Eclair 已经支持通道拼接，但这一变化使其更易于支持其他节点软件可能使用的通道拼接协议。{% assign timestamp="51:42" %}
+
+- [LND #7820][]将 `BatchOpenChannel` RPC 可用的所有字段添加到非批处理的 `OpenChannel` RPC 中，但除了 `funding_shim` (批量打开不需要)和 `fundmax` (打开多个通道时不能为一个通道提供所有余额)。{% assign timestamp="53:57" %}
+
+- [LND #7516][]使用新的 `utxo` 参数扩展了 `OpenChannel` RPC，该参数允许从钱包中指定一个或多个 UTXO，这些 UTXO 用于为新通道提供资金。{% assign timestamp="54:57" %}
+
+- [BTCPay Server #5155][]在后台添加了一个报告页面，可提供支付和链上钱包报告、导出到 CSV 的能力，并且可以通过插件来扩展。{% assign timestamp="57:26" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="863,26467,6378,6449,6399,6389,6403,6437,6398,5492,2680,7820,7516,5155" %}
+[clnrest doc]: https://github.com/rustyrussell/lightning/blob/02c2d8a9e3b450ce172e8bc50c855ac2a16f5cac/plugins/clnrest/README.md
+[news133 usdt]: /en/newsletters/2021/01/27/#bitcoin-core-19866
+[kc scripts]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004025.html
+[btcscripts spec]: https://btctranscripts.com/lightning-specification/
+[libera.chat]: https://libera.chat/
+[trevethan blind]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-July/021792.html
+[generalized blind schnorr]: https://gist.github.com/moonsettler/05f5948291ba8dba63a3985b786233bb
+[somsen gist]: https://gist.github.com/RubenSomsen/be7a4760dd4596d06963d67baf140406
+[lucre]: https://github.com/benlaurie/lucre
+[minicash]: https://github.com/phyro/minicash
+[cashu]: https://github.com/cashubtc/cashu
+[fiatjaf custodial]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004008.html
+[news210 commando]: /zh/newsletters/2022/07/27/#core-lightning-5370
+[dhke]: https://en.wikipedia.org/wiki/Diffie%E2%80%93Hellman_key_exchange
```

Comment:
I think the page on the Diffie-Hellman key exchange has a Chinese translation:
```suggestion
[dhke]: https://zh.wikipedia.org/wiki/%E8%BF%AA%E8%8F%B2-%E8%B5%AB%E7%88%BE%E6%9B%BC%E5%AF%86%E9%91%B0%E4%BA%A4%E6%8F%9B
```

Reply:
This is great! Thanks @murchandamus !